t/40-openqa-clone-job.t

```diff
@@ -32,7 +32,7 @@ sub test_once {
 }
 
 test_once '', qr/missing.*help for usage/, 'hint shown for mandatory parameter missing', 255, 'needs parameters';
-test_once '--help',        qr/Usage:/, 'help text shown',              1, 'help screen is not success';
+test_once '--help',        qr/Usage:/, 'help text shown',              0, 'help screen is not success';
```

Comment:
please also delete the word "not" in the explanation string, rest is fine. Self-approve and merge then please :)

Reply:
Done. But I cannot approve myself :)